examples/server/app/population.py

```diff
@@ -24,10 +24,8 @@
 gender_transform = CustomJSTransform(args=dict(source=ages), func="", v_func="""
     const val = new Float64Array(xs.length)
     for (let i = 0; i < xs.length; i++) {
-        if (source.data['Sex'][i] == 'Male')
-            val[i] = -xs[i]
-        else
-            val[i] = xs[i]
+        const sign = source.get("Sex")[i] == "Male" ? -1 : 1
```

Comment:
The old code still functions and this is just by way of improvement (here and other examples)?

Reply:
It did not. However, taking advantage of the fact that `ColumnDataSource.data` is wrapped in `PropertyValueColumnData`, I was able to add a special case to preserve compatibility. Otherwise in Python we don't know what is a true dict and what is a pseudo-struct encoded as dict. In bokehjs types and object identities are preserved, so that equality and mutability still work as expected.

Reply:
I reverted changes to bokeh's examples.